src/azure-cli/azure/cli/command_modules/alertsmanagement/_params.py

```diff
@@ -0,0 +1,182 @@
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+    target_resource_arg_type = CLIArgumentType(options_list=['--target-resource'], help='Filter by target resource id',
+                                               metavar='TARGET_RESOURCE')
+    target_resource_type_arg_type = CLIArgumentType(options_list=['--target-resource-type'],
+                                                    help='Filter by target resource type',
+                                                    metavar='TARGET_RESOURCE_TYPE')
+    target_resource_group_arg_type = CLIArgumentType(options_list=['--target-resource-group'],
+                                                     help='Filter by resource group',
+                                                     metavar='TARGET_RESOURCE_GROUP')
+    monitor_service_arg_type = CLIArgumentType(options_list=['--monitor-service'], help='Filter by target resource',
+                                               metavar='MONITOR_SERVICE')
+    monitor_condition_arg_type = CLIArgumentType(options_list=['--monitor-condition'],
+                                                 help='Filter by monitor condition. Supported values – Fired, Resolved',
+                                                 metavar='MONITOR_CONDITION')
+    severity_arg_type = CLIArgumentType(options_list=['--severity'],
+                                        help='Filter by severity. Supported values – Sev0, Sev1, Sev2, Sev3, Sev4',
+                                        metavar='SEVERITY')
+    state_arg_type = CLIArgumentType(options_list=['--state'],
+                                     help='Filter by alert state. Supported values - New, Acknowledged, Closed',
```

Comment:
could that be defined as enum?
If you define enum argument type, values would be included automatically in help.